packages/docs-theme/src/styles/_code-block.scss

```diff
@@ -5,4 +5,8 @@
 
 .#{$ns}-docs-code-block {
   overflow: auto;
+
+  .#{$ns}-dark & {
+    border: solid 1px $pt-dark-divider-white;
+  }
```

Comment:
this is the code in question we should consider adding to the core "typography" code block styles